frontend/pages/ManageControlsPage/Secrets/components/AddSecretModal/AddSecretModal.tsx

```diff
@@ -76,6 +76,8 @@ const AddSecretModal = ({ onCancel, onSave }: AddSecretModalProps) => {
       } finally {
         setIsSaving(false);
       }
+    } else {
+      setFormValidation(validation);
     }
```

Comment:
My refactor to have the modal do the API calls accidentally lost the "required" field validation error messages, this adds them back.  This was caught by my automated test 🙌